inc/classes/subscriber/Media/class-webp-subscriber.php

```diff
@@ -106,17 +106,17 @@ public function __construct( Options_Data $options_data, Options $options_api, C
 	 */
 	public static function get_subscribed_events() {
 		return [
-			'rocket_buffer'                                 => [ 'convert_to_webp', 30 ],
-			'rocket_cache_webp_setting_field'               => [
+			'rocket_buffer'                            => [ 'convert_to_webp', 16 ],
+			'rocket_cache_webp_setting_field'          => [
 				[ 'maybe_disable_setting_field' ],
 				[ 'webp_section_description' ],
 			],
-			'rocket_disable_webp_cache'                     => 'maybe_disable_webp_cache',
-			'rocket_third_party_webp_change'                => 'sync_webp_cache_with_third_party_plugins',
-			'rocket_homepage_preload_url_request_args'      => 'add_accept_header',
+			'rocket_disable_webp_cache'                => 'maybe_disable_webp_cache',
+			'rocket_third_party_webp_change'           => 'sync_webp_cache_with_third_party_plugins',
+			'rocket_homepage_preload_url_request_args' => 'add_accept_header',
 			'rocket_preload_after_purge_cache_request_args' => 'add_accept_header',
-			'rocket_preload_url_request_args'               => 'add_accept_header',
-			'rocket_partial_preload_url_request_args'       => 'add_accept_header',
+			'rocket_preload_url_request_args'          => 'add_accept_header',
+			'rocket_partial_preload_url_request_args'  => 'add_accept_header',
```

Comment:
Alignment issue.

Reply:
Believe it or not, PHP CS is happy with this, and was reporting an error before 🤷‍♂